src/gp-model/data/data.py

```diff
@@ -0,0 +1,76 @@
+import requests
+from datetime import datetime, timedelta
+from pymongo import MongoClient
+from config import configuration, environment
+
+
+def get_airqloud_sites(airqloud, tenant='airqo'):
+
+    # airqloud_url = 'https://platform.airqo.net/api/v1/devices/airqlouds'
+    params = {'tenant': tenant,
+              'airqloud': airqloud
+
+              }
+
+    if environment == 'staging':
+        headers = {'Authorization': configuration.API_TOKEN}
+        airqloud_response = requests.get(
+            configuration.VIEW_AIRQLOUD_URI, params=params, headers=headers)
+    else:
+        airqloud_response = requests.get(
+            configuration.VIEW_AIRQLOUD_URI, params=params)
+    airqloud_response_json = airqloud_response.json()
+
+    sites_details = airqloud_response_json['airqlouds'][0]['sites']
+
+    site_ids = [site['_id'] for site in sites_details]
+
+    return site_ids
+
+
+def get_site_data(site_id, tenant='airqo'):
+    start_time = (datetime.utcnow()-timedelta(days=7)
+                  ).strftime('%Y-%m-%dT%H:%M:%SZ'),
+    end_time = datetime.utcnow().strftime('%Y-%m-%dT%H:%M:%SZ')
+    event_url = 'https://platform.airqo.net/api/v2/devices/events'
```

Comment:
Hi @uman95, we can use an environment variable instead of hard coding `https://platform.airqo.net`

Reply:
Thanks @NoahNsimbe it was an oversight, thought I've cleaned it all up. New commit now available.